File: dart/lib/model/shipment.dart
part of api;


@Entity()
class Shipment {
  /* Unique identifier of service */
  String id = null;
  
  /* name of shipment */
  String name = null;
  
  /* priority of service, i.e. 1 = high, 2 = normal, 3 = low. default is 2. */
  int priority = null;
  
  
  Stop pickup = null;
  
  
  Stop delivery = null;
  
  /* array of capacity dimensions */
  List<int> size = [];
  
  /* array of required skills */
  List<String> requiredSkills = [];
  
  /* array of allowed vehicle ids */
  List<String> allowedVehicles = [];
  
  
  Shipment();

  @override
  String toString()  {
    return 'Shipment[id=$id, name=$name, priority=$priority, pickup=$pickup, delivery=$delivery, size=$size, requiredSkills=$requiredSkills, allowedVehicles=$allowedVehicles, ]';
  }

}

File: scala/src/main/scala/io/swagger/client/model/Shipment.scala
package io.swagger.client.model




case class Shipment (
  /* Unique identifier of service */
  id: String,
  /* name of shipment */
  name: String,
  /* priority of service, i.e. 1 = high, 2 = normal, 3 = low. default is 2. */
  priority: Integer,
  pickup: Stop,
  delivery: Stop,
  /* array of capacity dimensions */
  size: List[Integer],
  /* array of required skills */
  required_skills: List[String],
  /* array of allowed vehicle ids */
  allowed_vehicles: List[String])
  
